stats/src/main/java/io/airlift/stats/QuantileDigest.java

```diff
@@ -371,6 +376,91 @@ public boolean process(Node node)
         return builder.build();
     }
 
+    public double getCDF(long x) {
```

Comment:
CDF in stats nomenclature is almost like CPU in cs. Is that acceptable use?
The alternative is getCumulativeDistributionFunction(long x)